test/metabase/upload_test.clj

```diff
@@ -506,16 +514,17 @@
 (deftest load-from-csv-table-name-test
   (testing "Can upload two files with the same name"
     (mt/test-drivers (mt/normal-drivers-with-feature :uploads)
-      (mt/with-empty-db
-        (mt/with-current-user (mt/user->id :crowberto)
-          (let [csv-file-prefix (mt/random-name)
-                model-1 (upload-example-csv! :csv-file-prefix csv-file-prefix)
-                model-2 (upload-example-csv! :csv-file-prefix csv-file-prefix)]
-            (testing "tables are different between the two uploads"
-              (is (some? (:table_id model-1)))
-              (is (some? (:table_id model-2)))
-              (is (not= (:table_id model-1)
-                        (:table_id model-2))))))))))
+      (let [csv-file-prefix "some file prefix"]
+        (with-upload-table!
```

Comment:
Wrapping the creation of each table in `with-upload-table!` cleans up after the created table, making `mt/with-empty-db` unnecessary